src/functions-reference/binary_distributions.Rmd

```diff
@@ -147,64 +158,98 @@ dropping constant additive terms.
 
 `real` **`bernoulli_logit_glm_lpmf`**`(int y | matrix x, real alpha, vector beta)`<br>\newline
 The log Bernoulli probability mass of y given chance of success
-`inv_logit(alpha + x * beta)`, where the same intercept `alpha` and dependant variable value `y` are used
```

Comment:
I cut a lot of the text here about the size constraints. It's unnecessary I think. We aren't explicit about size constraints in other distributions.